R-package/R/lgb.Dataset.R

```diff
@@ -412,8 +412,8 @@ Dataset <- R6::R6Class(
 
         # Trying to work with unknown dimensions is not possible
```

Comment:
Left after `Dataset$dim()` too?
```suggestion
        # Trying to work with unknown formats is not possible
```

Reply:
blegh yes, definitely. Thank you! I shouldn't work on these things late at night 😆